src/consensus/validation.h

```diff
@@ -64,11 +64,12 @@ class CValidationState {
         return mode == MODE_ERROR;
     }
     bool IsInvalid(int &nDoSOut) const {
+        bool isInvalid = false;
```

Comment:
Replacing code which spans over two lines to spread over six lines can only decrease readability.